uspv/init.go

```diff
@@ -137,6 +119,54 @@ func (s *SPVCon) Connect(remoteNode string) error {
 		return err
 	}
 	s.WBytes += uint64(n)
+	return nil
+}
+
+// Connect dials out and connects to full nodes.
+func (s *SPVCon) Connect(remoteNode string) error {
+	var err error
+	if lnutil.YupString(remoteNode) {
+		// if remoteNode is "yes" but no IP specified, use DNS seed
+		if len(s.Param.DNSSeeds) == 0 {
+			// no available DNS seeds
+			return fmt.Errorf(
+				"Can't connect: No known DNS seeds for %s", s.Param.Name)
+		}
+		listofDNSSeeds := s.Param.DNSSeeds
+		for len(listofDNSSeeds) != 0 {
+			seed := listofDNSSeeds[0]
+			seedAdrs, err := s.GetSeedAdrs(seed)
+			if err != nil {
+				return err
+			}
+			err = s.DialNode(seedAdrs)
+			if err != nil {
+				listofDNSSeeds = listofDNSSeeds[1:]
+				continue
+			}
+			err = s.Handshake(seedAdrs)
+			if err != nil {
+				// means we either have a sapm node or didn't get a resonse. So we Try again
+				log.Println(err)
+				log.Println("Couldn't establish connection with node. Proceeding to the next one")
+				continue
+			}
+			break
+		}
+	} else { // else connect to user-specified node
```

Comment:
we don't need a separate codepath for a user-supplied address.  If we create a address slice with only the supplied address, the rest of the code should work the same, but with skipping only GetSeedAdrs()